drainer/translator/translator.go

```diff
@@ -44,37 +44,71 @@ func getParser() (p *parser.Parser) {
 	return
 }
 
-func insertRowToDatums(table *model.TableInfo, row []byte) (pk types.Datum, datums map[int64]types.Datum, err error) {
+func insertRowToDatums(table *model.TableInfo, row []byte) (datums map[int64]types.Datum, err error) {
 	colsTypeMap := util.ToColumnTypeMap(table.Columns)
 
-	// decode the pk value
-	var remain []byte
-	remain, pk, err = codec.DecodeOne(row)
-	if err != nil {
-		return types.Datum{}, nil, errors.Trace(err)
+	var (
+		commonPKInfo *model.IndexInfo
+		pkLen        = 1
+	)
+	if table.IsCommonHandle {
```

Comment:
is the feature is enable and there's no pk for the table, this is false?

Reply:
yes,  no pk table always be non-clustered index table now.

but maybe it should have a chance for someone to implement a new feature like "drop PK in clustered index table" or others(related to his impl) in the future, and if they produce a table like that... they will meet `errors.New("Unsupported clustered index without primary key")` error and fix that logic